### DISCUSSION TITLE
Upgradeable spawners support

### DISCUSSION THREAD

**Discussion by Muszek1** [BG-Software-LLC/SuperiorSkyblock2#1114] [Pending Feature Requests]
Could you add support for Upgradeable spawners?

**Comment by Lemvs** [comment]
@Muszek1 if you can share the plugin so I can test that kind of support after adding it, I can give it a try. I recently added support for [SmartSpawner](https://github.com/BG-Software-LLC/SuperiorSkyblock2/commit/80b2c029ccc38392f13e232e3754ef8118221586), so there shouldn't be any problems with that plugin either, I just need a way to test it.
